recipes/ushuffle/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: ushuffle
+  version: 1.0.0
+
+about:
+  home: http://digital.cs.usu.edu/~mjiang/ushuffle/
+  license: custom
+  summary: uShuffle---a useful tool for shuffling biological sequences while preserving the k-let counts
+
+build:
+  number: 0
+
+source:
+  - url: http://digital.cs.usu.edu/~mjiang/ushuffle/main.c
```

Comment:
I don't think this will work. You can only have one URL.
Moreover, these URLs do not seem to be versioned. As a last resort you can pack those and put them into https://github.com/galaxyproject/cargo-port